### DISCUSSION TITLE
No animal_mt seed reads found!

### DISCUSSION THREAD

**Discussion by DThuo** [Kinggerm/GetOrganelle#334] [Q&A]
**First check**
![image](https://github.com/Kinggerm/GetOrganelle/assets/89732702/68855165-3783-4b1f-adfd-5623f2492dc0)

I am getting a  No animal_mt seed reads found!  Please check your raw data or change your animal_mt seed!

I have the seed reads and my .fastq.gz raw data in the correct PATH. 

The Example data provided is running but my data is not.

**Comment by JianjunJin** [comment]
Your input and execution look good. 

Here, the `No animal_mt seed reads found` means that GetOrganelle cannot find any match between the default seed database and your input reads, which is very likely given your limited amount of reads input.
We usually expect much more data input (like a few Gb for animal mt), given the relatively low percentage of organelle genomes in the WGS dataset. The percentage varies by sample largely. The example dataset is made up of a high mt concentration just for testing.
